### PR TITLE
JavaScript function to check if a given string is a palindrome

### PR DESCRIPTION
**Pull request by SaharAbdelhaq**
JavaScript function to check if a given string is a palindrome

### CODE REVIEW

File: Excersises/exercise6.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/* Write a JavaScript function to check if a given string is a palindrome (reads the same forwards and backwards).*/
+
+const checkPalindrome = (str) => {
+  let righToleft = "";
+  let leftToRight = str;
```
Comment (Abdallah307) [inline Excersises/exercise6.js]: No need to store it in a variable

File: Excersises/exercise6.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+/* Write a JavaScript function to check if a given string is a palindrome (reads the same forwards and backwards).*/
+
+const checkPalindrome = (str) => {
+  let righToleft = "";
```
Comment (Abdallah307) [inline Excersises/exercise6.js]: Please refactor and rename it to (e.g: reversedStr)

File: Excersises/exercise6.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  let leftToRight = str;
+
+  for (let i = str.length; i >= 0; i--) {
+    righToleft += str.charAt(i);
```
Comment (Abdallah307) [inline Excersises/exercise6.js]: please access string character like this : str[i]

File: Excersises/exercise6.js (hunk 1 above)
Comment (SaharAbdelhaq) [reply Excersises/exercise6.js]: Ok, done

File: Excersises/exercise6.js (hunk 2 above)
Comment (SaharAbdelhaq) [reply Excersises/exercise6.js]: Ok, done

File: Excersises/exercise6.js (hunk 3 above)
Comment (SaharAbdelhaq) [reply Excersises/exercise6.js]: Not working!

**Pull request merged**